src/components/alert/alert.ts

```diff
@@ -15,6 +15,8 @@ import type { CSSResultGroup } from 'lit';
 const toastStack = Object.assign(document.createElement('div'), { className: 'sl-toast-stack' });
 
 /**
+ * Alerts are used to display important messages inline or as toast notifications.
```

Comment:
Did you have any luck with `@summary`?

To reiterate from above, maybe this should look like:

```js
/**
 * @name Alert
 * @summary Alerts are used to display important messages inline or as toast notifications.
 */
```